Resolve FIXME: Use AppKit constants instead of strings

LumioseSil opened the pull request:
#### 7b2588ddab67b4b3aac3c5965721505b551cfd4b
<pre>
Resolve FIXME: Use AppKit constants instead of strings
<a href="https://bugs.webkit.org/show_bug.cgi?id=267073">https://bugs.webkit.org/show_bug.cgi?id=267073</a>

Reviewed by Simon Fraser.

We can use the AppKit constants now that they have been available for a
long time.

* Source/WebKitLegacy/mac/WebView/WebView.mm:
  (-[WebView addWindowObserversForWindow:]): Use AppKit constants.
  (-[WebView _windowDidChangeBackingProperties:]): Ditto.

Canonical link: <a href="https://commits.webkit.org/272740@main">https://commits.webkit.org/272740@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1a4a035ec2d9832795b54d4c82a4ad4428e709fb

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/32569 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/11317 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/34417 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/35135 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/29441 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/13666 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/8511 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28965 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/33003 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/9538 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/29104 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/8314 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/8448 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/29045 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/36471 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/29601 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/29467 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/34571 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/8578 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/6531 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/32430 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/10253 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7632 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/9190 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/9171 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aprotyas reviewed (commented):
LGTM, but I'm not a reviewer, so you'll need a WebKit reviewer r+.

aprotyas reviewed (commented):
Can you update the Bugzilla title so that it matches your patch title?

LumioseSil committed:
Resolve FIXME: Use AppKit constants instead of strings
https://bugs.webkit.org/show_bug.cgi?id=267073

Reviewed by Simon Fraser.

We can use the AppKit constants now that they have been available for a
long time.

* Source/WebKitLegacy/mac/WebView/WebView.mm:
  (-[WebView addWindowObserversForWindow:]): Use AppKit constants.
  (-[WebView _windowDidChangeBackingProperties:]): Ditto.

Canonical link: https://commits.webkit.org/272740@main
